apps/private-law/prl-cos-cron-email-notification/prl-cos-cron-email-notification.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: helm.toolkit.fluxcd.io/v2beta1
+kind: HelmRelease
+metadata:
+  name: prl-cos-cron-email-notification
```

Comment:
this needs to be referenced somewhere if you want it to actually be deployed

Reply:
For now I've referenced under preview to verify, pls review @timja